src/test/kotlin/com/terraformation/backend/seedbank/db/accessionStore/AccessionStoreBagTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.terraformation.backend.seedbank.db.accessionStore
+
+import com.terraformation.backend.db.seedbank.AccessionId
+import com.terraformation.backend.db.seedbank.BagId
+import com.terraformation.backend.db.seedbank.tables.pojos.BagsRow
+import com.terraformation.backend.seedbank.model.AccessionModel
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class AccessionStoreBagTest : AccessionStoreTest() {
+  @Test
+  fun `bag numbers are not shared between accessions`() {
+    val payload = AccessionModel(bagNumbers = setOf("bag 1", "bag 2"), facilityId = facilityId)
+    store.create(payload)
+    store.create(payload)
+
+    val initialBags = bagsDao.fetchByAccessionId(AccessionId(1)).toSet()
+    val secondBags = bagsDao.fetchByAccessionId(AccessionId(2)).toSet()
+
+    Assertions.assertNotEquals(initialBags, secondBags)
+  }
+
+  @Test
+  fun `bags are inserted and deleted as needed`() {
+    val initial =
+        store.create(AccessionModel(bagNumbers = setOf("bag 1", "bag 2"), facilityId = facilityId))
+    val initialBags = bagsDao.fetchByAccessionId(AccessionId(1))
+
+    // Insertion order is not defined by the API, so don't assume bag ID 1 is "bag 1".
+
+    Assertions.assertEquals(
```

Comment:
This was probably IntelliJ's fault, but can we add static imports for all these assertion methods so we don't have to have `Assertions.` prefixes in all the test methods?

Reply:
oh dangit intelliJ, yup, i can definitely fix those